cfn-resources/project/cmd/resource/resource.go

```diff
@@ -189,7 +199,8 @@ func Update(req handler.Request, prevModel *Model, currentModel *Model) (event h
 	if currentModel.Profile == nil || *currentModel.Profile == "" {
 		currentModel.Profile = aws.String(profile.DefaultProfile)
 	}
-	client, pe := util.NewMongoDBClient(req, currentModel.Profile)
+	client, pe := util.NewAtlasClient(&req, currentModel.Profile)
+	atlasV2 := client.AtlasV2
 	if pe != nil {
 		_, _ = logger.Warnf("CreateMongoDBClient error: %v", *pe)
```

Comment:
if we don't need the result we can just do: logger.Warnf("CreateMongoDBClient error: %v", *pe)

Reply:
adjusted

Reply:
i see, ok, that's fine